spec/support/features/session_helper.rb

```diff
@@ -308,6 +308,12 @@ def click_continue
       click_button t('forms.buttons.continue') if page.has_button?(t('forms.buttons.continue'))
     end
 
+    def click_setup
```

Comment:
I think the name here might imply some potential for reuse for a common "setup" button label, but the label is pretty specific to the security key setup. I'd suggest either inlining `click_button t('forms.webauthn_setup.set_up')` where we use this method, or moving it into `webauthn_helper.rb`, or updating the name to reflect that this is specific to security key.